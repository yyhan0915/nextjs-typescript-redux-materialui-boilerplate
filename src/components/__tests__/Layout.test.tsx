import React from 'react';
import { mount } from 'enzyme';
import LayOut from '../Layout';

describe('LayOut page', () => {
    it('match snapshot', () => {
        const wrapper = mount(<LayOut />);
        expect(wrapper).toMatchSnapshot();
    });
});
