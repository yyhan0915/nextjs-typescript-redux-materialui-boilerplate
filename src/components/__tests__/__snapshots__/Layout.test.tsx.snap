// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayOut page match snapshot 1`] = `ReactWrapper {}`;
